### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kokobe**
* Adds some missing release building prerequisites
* Makes it more clear to run `scripts/build-release.sh` first
* Building release or staging from Android Studio is actually a lot more complicated. Requires Darin's help with the setup prior.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +127,8 @@ scripts/install-apks.sh release
 scripts/install-apks.sh staging
 ```
 
+If you don't have signing keys, then you can use the `staging` build variant in Android Studio. Within Android Studio, you can set the active Build Variant to `release` or `staging` and build/install the app from there. This might require extra steps so ask @darinwf for setup help.
```
Comment (dan-alcantara-neeva) [inline README.md:130]: You probably want to just run the scripts instead.  There's a `scripts/build-staging.sh` file that you can run to generate a staging build.

File: README.md [hunk 2]
```diff
@@ -91,14 +91,24 @@ Whenever you fetch updates from the repository, make sure that you re-run `boots
...
-Run `scripts/build-release.sh` to generate the app bundle `build/release/neeva.aab`.
+```
+# Ensure that you have the proper environment variables set up:
+# NEEVA_REPO = path/to/neeva
+# NEEVA_KEYSTORE_PATH = path/to/playstore_signing_key
+# NEEVA_KEYSTORE_PASS = playstore signing password
+```
+
+```
+# To generate the app bundle `build/release/neeva.aab`. If you only want to test release out, cancel out of the script when prompted if "Build uploaded to App/Play Store"
```
Comment (dan-alcantara-neeva) [inline README.md:107]: Clean up the formatting so the first sentence makes sense:

```
# To generate the app bundle `build/release/neeva.aab`, run:
scripts/build-release.sh

# If only want to test the build, kill the script when asked "Build uploaded to App/Play Store?"
```

File: README.md [hunk 3]
```diff
@@ -91,14 +91,24 @@ Whenever you fetch updates from the repository, make sure that you re-run `boots
...
+scripts/build-release.sh
+```
 
-Alternatively, within Android Studio, you can set the active Build Variant to  `release` or `staging` and build the app from there. To then assemble `neeva.aab`, you can run:
+To then assemble `neeva.aab`, you can run:
 
 ```
 # To make a release bundle:
```
[the remark this answers is not in the document]
Comment (dan-alcantara-neeva) [reply README.md:114]: They're run as part of the build script.

[the remark this answers is not in the document]
Comment (Kokobe) [reply README.md:114]: oh wait i should delete those steps, let me test that first

**Review by dan-alcantara-neeva — APPROVED** [review]

**Pull request merged**